src/Http/Controllers/Character/CharacterController.php

```diff
@@ -61,7 +62,7 @@ public function getCharactersData(Request $request)
             }) :
             $this->getAllCharactersWithAffiliations();
```

Comment:
You need to pass false as parameter of this function. Otherwise every things related to paginate, order or filter will be processed on the result of a full SQL query. Which is resulting in an heavy loading time.

Sending false to it will move average transaction time from 7 seconds to 2 seconds. Must make some tests at ACL levels in order to be sure making search will not remove applied filters.